Q_AND_A.md

```diff
@@ -0,0 +1,24 @@
+
+# DWN Q&A
+
+## Message ID (`messageCid`) and Record ID (`recordId`)
+
+- Why can't/don't we use the message ID (`messageCid`) of the initial `RecordsWrite` as the record ID?
+
+  Because the initial `RecordsWrite` contains the `recordId` property, so this would be a "chicken-and-egg" problem. See question [why is `recordId` needed in the `RecordsWrite`](#why-is-record-id-needed) for more info.
```

Comment:
For brevity, i think we can leave out the second sentence (`See question`). It's directly below so they'll just keep reading

Reply:
I added it in case the question gets edited/shuffled, and the continuation is lost thus answer isn’t providing sufficient context